Makefile

```diff
@@ -23,26 +27,18 @@ modules_install:
 	$(MAKE) -C $(KERNEL_SRC) V=1 M=$(PWD) modules_install
 
 #Performs a dkms install
-install: dkms-prepare
-	dkms install -m mtcadummy -v ${MTCADUMMY_PACKAGE_VERSION} -k $(KVERSION)
+kernel_install:
+	$(MAKE) -C $(KERNEL_SRC) V=1 M=$(PWD) install
 
-#Performs a dmks remove
-#Always returns true, so purge also works if there is no driver installed
-uninstall:
-	dkms remove -m mtcadummy -v ${MTCADUMMY_PACKAGE_VERSION} -k $(KVERSION) || true
+packaging_install: configure-source-files
```

Comment:
Doesn't this manually do what the `dkms install` command does?

Reply:
How?